### DISCUSSION TITLE
Precipitation probability for GFS05 Ensemble Model

### DISCUSSION THREAD

**Discussion by magheralex** [open-meteo/open-meteo#1044] [General]
Hi Patrick,

Which ensemble model can I use to retrieve precipitation probability? For example, downloading the gfs05_ens variables shown below does not return precipitation probability:

docker run --rm -v open-meteo-data:/app/data ghcr.io/open-meteo/open-meteo download-gfs gfs05_ens --only-variables temperature_2m,cloud_cover,precipitation,frozen_precipitation_percent,showers,cape,wind_gusts_10m,categorical_freezing_rain,visibility,lifted_index --run 0, including --second-flush

I specifically want to obtain precipitation probability for both gfs (05 or 025) and ecmwf_ifs025 ensemble models. I also tried to 
sync ecmwf_ifs025_ensemble precipitation_probability

_or_ 

docker run -it --rm -v open-meteo-data:/app/data ghcr.io/open-meteo/open-meteo download-ecmwf --domain ifs025_ensemble --only-variables precipitation,cloud_cover,relative_humidity_2m,cape --run 0

, but it does not work (got daily precipitation_probability_max" as [null]).

Thanks for your help!

**Comment by patrick-zippenfenig** [comment]
Ensemble domains for precipitation probability: `bom_access_global_ensemble ecmwf_ifs025_ensemble cmc_gem_geps dwd_icon_d2_eps dwd_icon_eps dwd_icon_eu_eps ncep_gefs025 ncep_gefs05`

Depending on the location, the API may select a different ensemble model. You might have to add all of them

**Reply by magheralex** [reply]
thanks but gfs05_ens does not return precipitation_probability. For example, if I download these parameters, I got null values for precipitation_probability:
docker run --rm -v open-meteo-data:/app/data ghcr.io/open-meteo/open-meteo download-gfs gfs05_ens --only-variables temperature_2m,cloud_cover,precipitation,frozen_precipitation_percent,showers,cape,wind_gusts_10m,categorical_freezing_rain,visibility,lifted_index --run 0

The same for ecmwf_ifs025:
docker run -it --rm -v open-meteo-data:/app/data ghcr.io/open-meteo/open-meteo download-ecmwf --domain ifs025_ensemble --only-variables precipitation,cloud_cover,relative_humidity_2m,cape --run 0

Do I need specific variables?

**Reply by patrick-zippenfenig** [reply]
They should emit precipitation probabilities just like this. Are you running the latest docker version? This feature was implemented just a couple of months ago

**Reply by magheralex** [reply]
Hi Patrick, 
You're right. I was running an older version. It works with latest version.
Thanks a lot for your help!
